src/spaceObjects/cpuShip.cpp

```diff
@@ -213,6 +214,20 @@ void CpuShip::orderDock(P<SpaceObject> object)
     this->addBroadcast(FVF_Friendly, "Docking to " + object->getCallSign() + ".");
 }
 
+void CpuShip::orderRepair()
+{
+    if (orders != AI_Repair)
+    {
+        previous_orders = orders;
+        orders = AI_Repair;
+    }
+}
+
+void CpuShip::orderResume()
+{
+    orders = previous_orders;
```

Comment:
From what I see, all AI orders involving objects have a failsafe, if they target a non-existing object, they fall back to the Roaming AI. So I don't think it's necessary. I will test that later and see if any there is any funky behavior to be sure.